allensdk/brain_observatory/behavior/behavior_session.py

```diff
@@ -157,7 +192,87 @@ def get_rolling_performance_df(self):
 
         return performance_metrics_df
 
-    def get_performance_metrics(self, engaged_trial_reward_rate_threshold=2):
+    def get_performance_metrics(
+            self,
+            engaged_trial_reward_rate_threshold: float = 2.0
+        ) -> dict:
+        """Get a dictionary containing a subject's behavior response
+        summary data.
+
+        Parameters
+        ----------
+        engaged_trial_reward_rate_threshold : float, optional
+            The number of rewards per minute that needs to be attained
+            before a subject is considered 'engaged', by default 2.0
+
+        Returns
+        -------
+        dict
+            Returns a dict of performance metrics with the following fields:
+                trial_count:
+                    The length of the trial dataframe
+                    (including all 'go', 'catch', and 'aborted' trials)
+                go_trial_count:
+                    Number of 'go' trials in a behavior session
+                catch_trial_count:
+                    Number of 'catch' trial types during a behavior session
+                hit_trial_count:
+                    Number of trials with a hit behavior response
+                    type in a behavior session
+                miss_trial_count:
+                    Number of trials with a miss behavior response
+                    type in a behavior session
+                false_alarm_trial_count:
+                    Number of trials where the mouse had a false alarm
+                    behavior response
+                correct_reject_trial_count:
+                    Number of trials with a correct reject behavior
+                    response during a behavior session
+                auto_rewarded_trial_count:
+                    Number of trials where the mouse received an auto
+                    reward of water.
+                rewarded_trial_count:
+                    NEEDS A NEW DESCRIPTION?
```

Comment:
@dougollerenshaw Since I've changed what this field returns, does it need a new description?

Reply:
Description should be: "rewarded_trial_count: Number of trials on which the animal was rewarded for licking in the response window"

Reply:
Updated.